.github/workflows/PullRequestCreated.yml

```
@@ -15,6 +15,8 @@ jobs:
         github.event.pull_request.head.repo.full_name == github.repository
         && github.event.sender.type != 'Bot'
         && ${{ !startsWith(github.event.pull_request.title, 'NO-JIRA') }}
```

```suggestion
        && !startsWith(github.event.pull_request.title, 'NO-JIRA')
```

I think we should fix the entire action at once